experimental/generated/beats/fields.ecs.yml

```diff
@@ -2699,7 +2699,7 @@
       default_field: false
     - name: trigger
       level: extended
-      type: nested
+      type: object
       description: Details about the function trigger.
       default_field: false
```

Comment:
Can we just remove the `trigger` field altogether? It's not really necessary as the `trigger.request_id` field already implicitly creates a `trigger` object. Explicitly mapping a field as an `object` is also problematic in combination with `subobjects: false`.

There's already precedent in ECS to not explicitly define object fields. For example:

https://github.com/elastic/ecs/blob/261a873fcb24ceb9e1a37f40d12905549197579c/schemas/log.yml#L67